hugegraph-test/src/main/java/com/baidu/hugegraph/core/VertexCoreTest.java

```diff
@@ -6667,6 +6668,37 @@ public void testQueryByPageWithInvalidPage() {
         });
     }
 
+
+    @Test
+    public void testQueryByPageWithSpecialBase64CharacterPage() {
+        // TODO: Not ready for commit, lack valid page contains '+' or '/'
```

Comment:
fiexed？

Reply:
fixed now, it's hard to get a **valid page** contains '+' '/' '=' ....